README.rst

```diff
@@ -230,8 +230,8 @@ A configuration file must be specified and used for the credentials to communica
 .. code-block:: ini
 
     [NEST]
-    user = joe@user.com
-    password = swordfish
+    client_id = yourclientid
```

Comment:
If this is not directly related to the streaming API, please submit changes into a pull request in the main repo.